src/components/canvas-tools/text-tool.sass

```diff
@@ -1,7 +1,7 @@
 @import ../vars
 
 .text-tool
-  width: 100%
+  width: calc(100% - 2px)
```

Comment:
Was wondering, is it preferable to have the Slate text region take up the whole height of the tile as well?